client/src/util/getLocalisedString.ts

```diff
@@ -0,0 +1,22 @@
+import forEach from "lodash/forEach";
+
+import { SUPPORT_LANGUAGES } from "../constants";
+import { LocalizedObject } from "../generated/graphql";
+import { Language } from "../types";
+
+export default (obj: LocalizedObject, language: Language) => {
+  const languages = [
+    language,
+    ...Object.values(SUPPORT_LANGUAGES).filter(item => item !== language)
+  ];
+  let str = "";
+
+  forEach(languages, lng => {
```

Comment:
[`Array.prototype.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) does exactly what you are doing here

Reply:
Also, a unit-test would be helpful documentation for this module